ee/tasks/replay.py

```diff
@@ -19,7 +24,17 @@
 # to much less than that
 @shared_task(ignore_result=False, queue=CeleryQueue.SESSION_REPLAY_EMBEDDINGS.value, rate_limit="75/m")
 def embed_batch_of_recordings_task(recordings: List[Any], team_id: int) -> None:
-    embed_batch_of_recordings(recordings, team_id)
+    try:
+        team = Team.objects.get(id=team_id)
+        runner = SessionEmbeddingsRunner(team=team)
+
+        runner.run(recordings, embeddings_preparation=SessionEventsEmbeddingsPreparation)
+
+        results = fetch_errors_by_session_without_embeddings(team.pk)
+        runner.run(results, embeddings_preparation=ErrorEmbeddingsPreparation)
```

Comment:
I'd be tempted to pass in the preparation in the constructor since the runner is invalid without it and then you'd have `BlahRunner(team, preparation).run(recordings)` but that's very nit-picky